tests/unit/channel_tests.py

```diff
@@ -330,7 +334,7 @@ def test_basic_consume_consumers_rpc_with_no_completion_callback(self, rpc, _unu
         expectation = spec.Basic.Consume(
             queue='test-queue',
             consumer_tag=consumer_tag,
-            auto_ack=False,
+            no_ack=False,
```

Comment:
The programmatically-generated code in pika/spec.py uses the AMQP `no_ack` versus the new name `auto_ack` that we adopted recently in the connection adapters.